requirements/bddtests.txt

```diff
@@ -34,6 +34,8 @@ billiard==3.6.4.0
     # via
     #   -r requirements/requirements.txt
     #   celery
+build==0.8.0
+    # via pip-tools
```

Comment:
The various new dependencies that're being added to the `*.txt` files are due to `pip-tools` having been added.